backend/main.py

```
@@ -139,19 +163,19 @@
 
+# Health monitoring router
+try:
+    from database_self_healing_system import router as health_router
+    HEALTH_ROUTER_AVAILABLE = True
+    print("✅ Health monitoring router available")
+except ImportError:
+    HEALTH_ROUTER_AVAILABLE = False
+    print("⚠️ Health monitoring router not available")
+
 # Import database initialization
 from init_database import initialize_jyotiflow_database
```

Ruff (0.12.2)

`init_database.initialize_jyotiflow_database` imported but unused

Remove unused import: `init_database.initialize_jyotiflow_database`

(F401)